Tests/fixtures/podcasting.yml

```diff
@@ -8554,7 +8575,7 @@
     platform: ""
   client:
     type: mobile app
-    name: Shadow
+    name: Shadowrocket
```

Comment:
No, this one was correct: there is a Shadow app for iOS https://apps.apple.com/us/app/podcast-player-shadow/id940127690

Reply:
I agree, I split these into their own entries.